MovieCave - MVVM/MovieCave/MovieCave/Coordinators/MoviesViewCoordinator.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  MoviesViewCoordinator.swift
+//  MovieCave
+//
+//  Created by Admin on 28.09.23.
+//
+
+import UIKit
+
+protocol MoviesViewCoordinatorDelegate: AnyObject {
+    
+    /// Loads the MovieDetailsView and ViewModel to display additional
+    /// information for a selected movie.
+    /// - Parameter ID: The ID of the selected movie to load details for.
+    func loadMoviesDetailsView(with movieID: Int)
```

Comment:
Typo here. Should be loadMovie... because you are loading the details of a single movie